test/utils/math/Math.test.js

```diff
@@ -710,4 +711,118 @@ describe('Math', function () {
       });
     });
   });
+
+  describe('reverseBits', function () {
+    describe('reverseBitsUint256', function () {
+      it('reverses bytes correctly', async function () {
+        await expect(this.mock.$reverseBitsUint256(0)).to.eventually.equal(0n);
+        await expect(this.mock.$reverseBitsUint256(ethers.MaxUint256)).to.eventually.equal(ethers.MaxUint256);
+
+        // Test simple pattern
+        await expect(
+          this.mock.$reverseBitsUint256('0x0000000000000000000000000000000000000000000000000000000000000001'),
```

Comment:
IMO the uint256 and uint128 test pattern should contain more data than "all zero with a one". I like the uint64 test case